doc/LanguageClient.txt

```diff
@@ -358,7 +358,13 @@ supported. Check by running >
 Default: 1 when a floating window is supported, otherwise 0
 Valid Options: 1 | 0
 
-2.26 g:LanguageClient_usePopupHover           *g:LanguageClient_usePopupHover*
+2.27 g:LanguageClient_floatingHoverHighlight *g:LanguageClient_floatingHoverHighlight*
+
+Control how floating window hover highlights are displayed.
+
+Default: CursorLine
```

Comment:
It might be more convenient to allow user specify the winhl themselves, i.e., values can be `Normal:MyNormal,NormalNC:MyNormalNC`